checker/src/main/java/org/checkerframework/checker/nullness/NullnessTransfer.java

```diff
@@ -357,21 +358,35 @@ public TransferResult<NullnessValue, NullnessStore> visitMethodInvocation(
             MethodInvocationNode n, TransferInput<NullnessValue, NullnessStore> in) {
         TransferResult<NullnessValue, NullnessStore> result = super.visitMethodInvocation(n, in);
 
-        // Make receiver non-null.
+        MethodInvocationTree tree = n.getTree();
+        ExecutableElement method = TreeUtils.elementFromUse(tree);
+        boolean nonNullAssumption =
```

Comment:
Make this a field and initialize it once in the constructor.
What is a good field name? The local var name now isn't very specific. The option name is rather long. Ideas?

Reply:
We might not need to use this flag somewhere else (`visitMethodAccess` may not need it, see below), what's the use to make it as a field?

In the particular, we have a context, so I think it's a both specific and short name.

Reply:
`Transfer` function maybe invoked several times, it's good to store the value into field